metaflow/plugins/aws/batch/batch_client.py

```diff
@@ -149,7 +149,12 @@ def _register_job_definition(
         max_swap,
         swappiness,
         inferentia,
+        memory,
         host_volumes,
+        use_tmpfs,
+        tmpfs_tempdir,
+        tmpfs_size,
+        tmpfs_path,
```

Comment:
`tmpfs_path` needs to be an absolute path (start with `/`) or Batch barfs:
```
botocore.errorfactory.ClientException: An error occurred (ClientException) when calling the RegisterJobDefinition operation: Error executing request, Exception : The containerPath (foobar) specified for tmpfs is invalid. It must contain an absolute path., RequestId: de9d0a90-358c-48b3-b104-99d040595ed9
```
we could have a static check that ensuring
```
if tmpfs_path and tmpfs_path[0] != '/':
    raise BatchJobException("tmpfs_path needs to be an absolute path")
```
so the user doesn't have to wait until the task executes to see the error

Reply:
diff doesn't show "outdated" for this file but this is now covered after the latest commit. Check was added to the `step_init` in `batch_decorator.py`. I tried raising the exception inside the `batch_client.py` `_register_job_definition()` at first, but this was too late in the lifecycle and allowed the flow to start executing before encountering the exception.